code/core/src/test/java/com/adobe/marketing/mobile/services/ui/notification/AEPPushTemplateTests.kt

```diff
@@ -0,0 +1,365 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.services.ui.notification
+
+import androidx.core.app.NotificationCompat
```

Comment:
I haven't reviewed the test class assuming that there are more to be added. We can add them in a follow up PR if you think this is going to be very large

Reply:
yes i will add the tests / update these tests in a follow up PR